.gitignore

```diff
@@ -81,3 +81,4 @@ Makefile.in.in
 /tests/rpmtests.dir
 /tests/rpmtests.log
 /tests/testing/
+tests/data/scripts_pythondistdeps/usr/
```

Comment:
For consistency.

```suggestion
/tests/data/scripts_pythondistdeps/usr/
```

Reply:
Oh thank you, that would have bothered me if I realized it :D